python/cuxfilter/charts/datashader/datashader.py

```diff
@@ -104,6 +104,8 @@ def scatter(
     A cudashader scatter plot.
     Type cuxfilter.charts.datashader.custom_extensions.InteractiveImage
     """
+    if isinstance(color_palette, tuple):
+        color_palette = list(color_palette)
```

Comment:
Can `Scatter`, `Graph` etc be made indifferent to the type of sequence a palette is stored in?

Reply:
it's actually datashader that throws an error  [example](https://github.com/AjayThorve/cuxfilter/blob/new_dtypes/python/cuxfilter/charts/datashader/plots.py#L223) if the input color_palette is not a list, so we need to make sure it is a list during chart init.

Reply:
![image](https://user-images.githubusercontent.com/20476096/97906790-1cc46700-1cf9-11eb-8718-c1ab76e83b9b.png)

Reply:
That's a bit surprising, since I thought datashader could handle Bokeh palettes directly, but I guess not. But `Scatter` and `Graph` could do this check and update internally then? I'm asking about insulating callers of `Scatter` and `Graph` from the burden of having to know to massage the palettes in to some specific different format